### PR TITLE
LUN-124: Check modules enough words

### PR DESCRIPTION
**Pull request by Saraharar98**
This pull request belongs to an issue on our [bugtracker](https://issues.tuerantuer.org/secure/RapidBoard.jspa?rapidView=27&view=detail).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword _LUN_.

### CODE REVIEW

File: src/routes/ExercisesScreens.tsx [hunk 1]
```diff
@@ -94,10 +94,14 @@ const ExercisesScreen = ({ route, navigation }: ExercisesScreenPropsType): JSX.E
   }
 
   const handleNavigation = (item: ExerciseType): void => {
-    setSelectedKey(item.key.toString())
-    navigation.navigate(EXERCISES[item.key].nextScreen, {
-      discipline: discipline
-    })
+    if (item.title === 'Level 1' && numberOfChildren < 4) {
```
Comment (SteffiStuffel) [inline src/routes/ExercisesScreens.tsx:97]: Please check if the `item.title` equals the constant from the labels.
(import labels-file and check for labels.exercises.wordChoice.title)

File: src/routes/ExercisesScreens.tsx [hunk 2]
```diff
@@ -94,10 +94,14 @@ const ExercisesScreen = ({ route, navigation }: ExercisesScreenPropsType): JSX.E
...
+      Alert.alert('Not enough words')
```
Comment (SteffiStuffel) [inline src/routes/ExercisesScreens.tsx:98]: Please also don't use this here as a string directly, but add a entry to the labels file, with a speaking name. German translation for the alert message should be: 'Die Übung ist für dieses Modul nicht möglich (zu wenig Wörter)'

**Review by SteffiStuffel — CHANGES_REQUESTED** [review]
Looks already good. :rocket: 
Just some small changes needed.

File: src/constants/labels.json [hunk 3]
```diff
@@ -33,7 +33,8 @@
     },
     "wordChoice": {
       "title": "Level 1",
-      "description": "Das richtige Wort auswählen"
+      "description": "Das richtige Wort auswählen",
+      "speaking": "Die Übung ist für dieses Modul nicht möglich (zu wenig Wörter)"
```
Comment (SteffiStuffel) [inline src/constants/labels.json:37]: oke sorry for the confusion, the name should not be `speaking`, i meant it should be a meaningful name. e.g. `errorWrongModuleSize`

**Review by klinzo — APPROVED** [review]
Good to merge your first PR :tada:

**Comment by Saraharar98** [comment]
RE-open PR

File: src/routes/ExercisesScreens.tsx [hunk 4]
```diff
@@ -94,10 +94,14 @@ const ExercisesScreen = ({ route, navigation }: ExercisesScreenPropsType): JSX.E
   }
 
   const handleNavigation = (item: ExerciseType): void => {
-    setSelectedKey(item.key.toString())
-    navigation.navigate(EXERCISES[item.key].nextScreen, {
-      discipline: discipline
-    })
+    if (item.title === labels.exercises.wordChoice.title && numberOfChildren < 4) {
```
Comment (Taggotty) [inline src/routes/ExercisesScreens.tsx:97]: labels isn't imported and it should be `discipline.numberOfChildren` I think.

**Review by Taggotty — APPROVED** [review]
Tested on android emulator. Nice PR 🎉

**Pull request merged**